### PR TITLE
Generate triples for SVG hierarchy.

### PR DESCRIPTION
**Pull request by keyurva**
* Only generating triples in this PR. Will insert them into DB in the next one.
* Sorting by SVG IDs in the main code itself (vs in tests) so that both SVG json and triples are similarly sorted.

### CODE REVIEW

File: simple/stats/schema_constants.py [hunk 1]
```diff
@@ -28,7 +28,13 @@
 
 PREDICATE_TYPE_OF = "typeOf"
 PREDICATE_POPULATION_TYPE = "populationType"
+PREDICATE_MEMBER_OF = "memberOf"
+PREDICATE_SPECIALIZATION_OF = "specializationOf"
+PREDICATE_NAME = "name"
+
 TYPE_STATISTICAL_VARIABLE = "StatisticalVariable"
+TYPE_STATISTICAL_STAT_VAR_GROUP = "StatVarGroup"
```
Comment (shifucun) [inline simple/stats/schema_constants.py:36]: TYPE_STATISTICAL_VARIABLE_GROUP ?

**Review by shifucun — APPROVED** [review]
very nice!

File: simple/stats/schema_constants.py (hunk 1 above)
Comment (keyurva) [reply simple/stats/schema_constants.py]: Done.

**Comment by keyurva** [comment]
Thanks for the review!

**Pull request merged**